scripts/generate_user_space_activities_counters.ts

```diff
@@ -0,0 +1,38 @@
+import 'dotenv/config';
+import { refreshProposalsCount, refreshVotesCount } from '../src/helpers/actions';
+import db from '../src/helpers/mysql';
+
+// Usage: yarn ts-node scripts/generate_user_space_activities_counters.ts
+async function main() {
+  const spaces = await db.queryAsync('SELECT id, name FROM spaces');
+
+  for (const index in spaces) {
+    console.log(
+      `Processing space #${spaces[index].id} (${spaces[index].name}) - ${index + 1}/${spaces.length}`
+    );
+
+    const votesCountRes = await refreshVotesCount(spaces[index].id);
```

Comment:
But on delete-proposal we call it with an array `refreshVotesCount([msg.space]);`